build.sbt

```diff
@@ -100,8 +97,11 @@ scalacOptions in (Compile, console) ~= (_.filterNot(Set(
   "-Ywarn-unused-import",
   "-Xfatal-warnings"
 )))
+Global / onChangedBuildSource := ReloadOnSourceChanges
+
+Test / parallelExecution := false
 
-parallelExecution in Test := false
+ThisBuild / turbo := true
```

Comment:
Why should we enable this? From the description:

>sbt 1.3.0 adds “turbo” mode that enables experimental or advanced features that might require some debugging by the build user when it doesn’t work. 

The advanced features might be useful, but I'm a bit cautious on opening the door to any experimental features

Reply:
One of most important things it changes is classloader layering. Enabling turbo mode didn't broke anything so my preference is to keep it enabled and disable it only when it does break something for us.